lisa/tests/scheduler/load_tracking.py

```diff
@@ -345,6 +359,7 @@ def get_simulated_pelt(self, task, signal_name):
             clock = None
 
         try:
+            cpus = trace.analysis.tasks.cpus_of_tasks([task])
```

Comment:
Would be best to use a different variable name than `cpus` to avoid confusion with the previous definition. Maybe the first one could be renamed "pinned_cpus"

Reply:
OK, let me change this.